src/shared/components/ResourceEditor/ResourceEditor.less

```diff
@@ -3,9 +3,12 @@
 @import '../../../../node_modules/codemirror/addon/fold/foldgutter.css';
 @import '../../../../node_modules/codemirror/theme/base16-light.css';
 
-// codemirror styling
-.cm-s-base16-light span.cm-string {
-  // string values
+// CodeMirror styling
```

Comment:
I wasn't sure if this was a reference for future editor customization. I simply commented them out so that nothing is selected. But I think we can just remove them.